.azure-pipelines/workbooksValidations.yaml

```diff
@@ -0,0 +1,12 @@
+jobs:
+- job: "WorkbooksValidations"
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+  - task: Npm@1
+    displayName: 'npm install'
+    inputs:
+      verbose: false
+      command: 'install'
+  - script: 'npm run tsc && node .script/workbooksMetadataValidator.js'
```

Comment:
Please note that you add reference only to "workbooksMetadataValidator.js", means the other validations (like uniqueWorkbookKeyValidator) won't run.

you need to add here links to the other script as well.

Reply:
good point. Those are actually utility functions used by the `workbooksMetadataValidator`. I renamed them to `checkers` to prevent future confusion.